mkemlog.c

```diff
@@ -33,11 +32,20 @@ int main(int argc, char** argv) {
     char* file;
     char* number;
     char* end_of_number;
+    int emlog_max_size;
     uid_t uid = -1;
     if (argc < 2 || argc > 5) {
         error(1 ,0, USAGE);
     }
     file = argv[1];
+
+    FILE *max_size_file = NULL;
+    max_size_file = fopen("/sys/module/emlog/parameters/emlog_max_size", "r");
+    if (errno)
+        error(1, errno, "Emlog module not loaded\n");
+    fscanf(max_size_file, "%d", &emlog_max_size);
+    if (errno)
```

Comment:
Looking at `fscanf()` man page, it looks like, it doesn't set `errno` in all failure cases.
(for example, reading an empty file will leave `emlog_max_size` uninitialized, but I don't think `errno` would be set in this case)
Thus, I'd also simply check the return code (it should be 1 since we have only one variable to assign):
```
rc = fscanf(...);
if (rc != 1) {
    error(...);
}